.github/workflows/send-transactions-test.yml

```diff
@@ -0,0 +1,586 @@
+name: Test commit
+
+on:
+  pull_request: {}
+  push:
+    branches:
+      - develop
+      - hotfix/*
+      - release/*
```

Comment:
do we want also to run it on `main` branch and releases (versioned tags)?

Reply:
I think we can add the main branch too, I'll do that